### ISSUE TITLE
HuskHomesGUI - 

### ISSUE THREAD

**Issue by RhythmicSys** [WiIIiam278/HuskHomes2#298]
When editing a home or warp, the GUI item gets set back to the default of stone. 
This only happens on editing description or name, not location

Steps to reproduce:
1. Set a home or warp
2. Set an icon
3. Rename or change the description
4. The icon will have changed back to stone

#### Changing the description 
![Screenshot from 2023-01-10 12-15-08](https://user-images.githubusercontent.com/45906780/211653408-e5cfd917-2439-4a27-a22d-511468a20780.png)
![Screenshot from 2023-01-10 12-15-32](https://user-images.githubusercontent.com/45906780/211653412-8523b037-3191-44c8-a55c-7be3430531cf.png)

#### Changing the name (capitalization)
![Screenshot from 2023-01-10 12-16-13](https://user-images.githubusercontent.com/45906780/211653413-48365f3c-b1ab-4e56-9392-f563ee12113d.png)
![Screenshot from 2023-01-10 12-16-38](https://user-images.githubusercontent.com/45906780/211653414-05e8a82c-5047-469e-a3fd-a3180990535b.png)

`HuskHomes version 3.2.1`
`HuskHomesGUI version 2.0`
`git-Purpur-1888 (MC: 1.19.3)`

**Comment by ApliNi** [comment]
Hey, this problem has been fixed by me. It's in Pull requests. 
But I think it needs to be fixed in HuskHomes. 
```java
// SavedPositionMenu.java

// On line 272 and 308
// 然后更新一遍图标, 临时解决 https://github.com/ApliNi/HuskHomesGUI/issues/5
setPositionMaterial(position, item.getType());
```

**Comment by WiIIiam278** [comment]
fixed in 2.1
